doc/api/http.md

```diff
@@ -1458,6 +1458,15 @@ Returns `true` if the entire data was flushed successfully to the kernel
 buffer. Returns `false` if all or part of the data was queued in user memory.
 `'drain'` will be emitted when the buffer is free again.
 
+### response.writableFinished
```

Comment:
Alphabetically, this section needs to be placed before `### response.write(chunk[, encoding][, callback])`.

Reply:
#28681